### DISCUSSION TITLE
Diann v 1.9.2 - Linux

### DISCUSSION THREAD

**Discussion by Sophia-006** [vdemichev/DiaNN#1275] [General]
Hi Vadim,

Are there any changes to the code for Linux with the new version? Unfortunately, my code works perfectly with version 1.8.1, but with version 1.9.2, it doesn't recognise the FASTA file:

WARNING: protein inference is enabled but no FASTA provided - is this intended?
0 files will be processed
The following warnings or errors (in alphabetic order) were detected at least the indicated number of times:
WARNING: protein inference is enabled but no FASTA provided - is this intended? : 1
Finished

My code includes --fasta "/databases/XX.fasta". Could there be a requirement related to the order of parameters?

**Comment by vdemichev** [comment]
Hi Sophia,

Can you please share the full log? If it cannot load the FASTA, there's likely some issue with read permissions on Linux.

Best,
Vadim

**Comment by Sophia-006** [comment]
Hi Vadim, 
The full log is: 
Loading diann/1.9.2
  Loading requirement: squashfuse/0.5.0 apptainer/1.3.3
DIA-NN 1.9.2 (Data-Independent Acquisition by Neural Networks)
Compiled on Oct 31 2024 04:27:44
Current date and time: Mon Nov 25 11:29:14 2024
Logical CPU cores: 64
WARNING: protein inference is enabled but no FASTA provided - is this intended?

0 files will be processed

The following warnings or errors (in alphabetic order) were detected at least the indicated number of times:
WARNING: protein inference is enabled but no FASTA provided - is this intended? : 1
Finished

with version 1.9.1:
Loading diann/1.9.1
 Loading requirement: squashfuse/0.5.0 apptainer/1.3.3
DIA-NN 1.9.1 (Data-Independent Acquisition by Neural Networks)
Compiled on Jul 15 2024 09:42:01
Current date and time: Mon Nov 25 11:25:08 2024
Logical CPU cores: 64
WARNING: protein inference is enabled but no FASTA provided - is this intended?

0 files will be processed                                            

* I tried changing the script to include --no-prot-inf, but still the error seems to be the fasta.

**Comment by Sophia-006** [comment]
Hi Vadim, 
It seems it was a mistake on our end when installing DIA-NN 1.9.2. It's working fine. Thanks for your time. 

Cheers, 
Sophia
